frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1149,153 +1531,86 @@ function ReaderFooter:addToMainMenu(menu_items)
                 end
             },
             {
-                text = _("Show footer separator"),
+                text = _("Hide empty items"),
+                help_text = _([[This option will hide values like 0 or off.]]),
```

Comment:
Sounds misleading.

And yes, I know I'm also commenting on things that always were. ;-)